dfusion_rust_core/src/util.rs

```diff
@@ -0,0 +1,42 @@
+use std::str::FromStr;
+use web3::types::U256;
+
+pub fn u128_to_u256(x: u128) -> U256 {
+    U256::from_big_endian(&x.to_be_bytes())
+}
+
+pub fn u256_to_u128(x: U256) -> u128 {
+    u128::from_str(&x.to_string()).unwrap()
+}
+
+#[cfg(test)]
+pub mod tests {
+    use super::*;
+
+    #[test]
+    fn test_u128_to_u256() {
+        assert_eq!(
+            u128_to_u256(u128::max_value()),
+            U256::from_dec_str("340282366920938463463374607431768211455").unwrap(),
+            "failed on 128::max_value()"
+        );
+        assert_eq!(u128_to_u256(1u128), U256::from(1), "failed on 1u128");
+        assert_eq!(u128_to_u256(0u128), U256::from(0), "failed on 0u128");
+    }
```

Comment:
I have definitely seen this all the code contained in this file before... was it renamed?

Reply:
See above, moved from `driver/src/util.rs` to `dfusion_rust_core/src/util.rs`